client/src/pages/UserPage/UserPage.js

```diff
@@ -26,29 +25,58 @@ const preloaderSX = {
 
 const UserPage = () => {
   const { userData, error, isLoading } = useSelector((store) => store.userReducer);
+  const [value, setValue] = useState('1');
+
   const dispatch = useDispatch();
 
+  const handleChange = (event, newValue) => {
+    setValue(newValue);
+  };
+
   useEffect(() => {
     dispatch(fetchUserInfo());
   }, []);
 
   return (
-    <Container component={Paper} sx={containerSX}>
-      {isLoading && <CircularProgress sx={preloaderSX} />}
-      {userData && (
-        <>
-          <UserHeader userData={userData} />
-          <UserList userData={userData} />
-          <ButtonContainer />
-        </>
-      )}
-      {error && (
-        <Container>
-          <Typography variant="h2">Error code: {error.status}</Typography>
-          <Typography variant="h3">{error.statusText}</Typography>
-          <Typography>Please try again later</Typography>
-        </Container>
-      )}
+    <Container>
+      <Container component={Paper} sx={containerSX}>
+        {isLoading && <CircularProgress sx={preloaderSX} />}
+        {userData && (
+          <>
+            <UserHeader userData={userData} />
```

Comment:
```suggestion
              <Typography variant="h2" align="center" sx={{ my: '30px', fontSize: '30px' }}>
              {userData.firstName} {userData.lastName}
            </Typography>
```

isnt it easier to delete for example the whole userHeader component as it is useless and not needed and substitute it is with a simple typo component?

Reply:
agreed, done